packages/trader/src/AppV2/Components/RiskManagementItem/risk-management-item.scss

```diff
@@ -18,9 +17,32 @@
             all: unset;
         }
     }
-
     .deal-cancellation-badge {
         background-color: var(--semantic-color-monochrome-surface-normal-lowest);
         border-radius: var(--semantic-borderRadius-sm);
+
+        .dc-remaining-time {
+            color: var(--semantic-color-monochrome-textIcon-normal-high);
+            font-size: var(--core-fontSize-50);
+        }
+    }
+    &__action-sheet-content {
+        height: 23.6rem;
```

Comment:
Is there a need to give fixed height to the action-sheet content?

Reply:
yes, as per the below requirement, it should be a fixed value, same like for Stake bottom sheet, for example:
'_Take profit & Stop loss:
✅ Increase action sheet height to 400px fixed across all devices_'